src/main/java/com/tabnine/binary/fetch/BootstrapperSupport.java

```diff
@@ -0,0 +1,73 @@
+package com.tabnine.binary.fetch;
+
+import com.intellij.openapi.diagnostic.Logger;
+import com.intellij.util.text.SemVer;
+import com.tabnine.general.StaticConfig;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.prefs.Preferences;
+
+import static java.lang.String.format;
+
+public class BootstrapperSupport {
```

Comment:
This name is too generic and encourages others to add more methods to this class. How about `BootstraperVersions`?

Reply:
there is one version